webrtc-stats.html

```diff
@@ -325,19 +325,32 @@
         <dl class="idl" title="dictionary RTCMediaStreamStats : RTCStats">
           <dt>DOMString streamIdentifier</dt>
           <dd><p><code>stream.id</code> property</p></dd>
-          <dt>sequence<DOMString> trackIds</dt>
+          <dt>sequence&lt;DOMString&gt; trackIds</dt>
           <dd><p>This is the id of the stats object, not the
             <code>track.id</code>.</p></dd>
         </dl>
 
         <section id="mststats-dict*">
           <h3><dfn>RTCMediaStreamTrackStats</dfn> dictionary</h3>
+	  <p>An RTCMediaStreamTrackStats object represents the stats about a
+	  MediaStreamTrakc's connection to the PeerConnection object for which
+	  one calls getStats.</p>
+	  <p>It appears in the stats as soon as it is attached. If it is
+	    detached, it continues to appear, but with the "detached" member
+	    set to True.</p>

```

Comment:
oops missed the reference to PC before. However, should the property be set to True if it is detached from a RtpSender/RtpReceiver?